nni/experiment/config/utils/internal.py

```diff
@@ -198,3 +203,33 @@ def get_ipv4_address() -> str:
     addr = s.getsockname()[0]
     s.close()
     return addr
+
+def load_experiment_config(config_json: dict) -> ExperimentConfig | RetiariiExeConfig:
+    # avoid circular import
+    from nni.nas.experiment.config import RetiariiExeConfig
+    from nni.nas.experiment.pytorch import RetiariiExperiment
```

Comment:
Same here. If this is imported, when users try to resume **any** experiment, they must have pytorch installed.

Reply:
updated